### ISSUE TITLE
Ignoring files outside basepath (#34/#36) breaks archiving program

### ISSUE THREAD

**Issue by derobert** [Parchive/par2cmdline#64]
I recently upgraded par2cmdline (as part of upgrading to Debian Jessie). Now my video archiver CD/DVD/BR program no longer works.

In short, there is a video directory with video files sorted into subdirectories. The program creates a temporary directory and tells par2 to create a par2 file in the temporary directory, using input files in the video directory. This *used* to work, and saves a lot of I/O in not having to copy the files to the temporary directory. (Especially when the videos and temporary directories are both over NFS!). `genisoimage` puts everything together into one ISO. (The archiver program actually does a couple other things too, but they don't involve par2).

**Comment by BlackIkeEagle** [comment]
Could you give an example of the command ? And this is a duplicate of #61.

**Comment by derobert** [comment]
I'm not sure this is the same as #61, that one seems to want to keep the files separate—I'm not, I'm just generating them separately, once the ISO is generated they sit in the same directory as the source files.

Example command line (run inside the video directory): 

    par2 c -v -m256 -r10 -- /srv/videos/tmp/udf-gq1NyRs5/mnt/zz_recover/CD subdir1/video1.mkv subdir1/video2.mkv subdir2/video3.mkv ….

Ultimately, that'll wind up on the disc with all of the files flat in `zz_recover/`

**Comment by BlackIkeEagle** [comment]
Yeah its the same as #61 since you initially want to generate the par2 files oustide the 'basepath'

**Comment by dermoth** [comment]
+1... Actually I worked around this by using symlinks, but it adds extra steps to my archival tool. That'S also with symlinks that I can recover data too (as I normally keep the structure, though there are no duplicate file names)

Out of curiosity, what prevents par2 from using full paths? I was there when the initial par utility was created for NNTP file transfer, and I get why it ended up so complex, although I bet nowadays most users ate merely creating redundancy data for long-term archival which is quite different. I wish the tool could better support this use case.

On a side note these seem to have been some work toward a "par3" container/tool which solve tree and speed/scalability issues (https://multipar.eu/), but so far the code is for windows only (I imported the code to git, fixed line-endings and converted to UTF-8, but most comments are in Japanese... I'm not sure I can find the time to port it to Linux but I'll share the repo if anyone's interested).
